File: python/1013.py
# -*- coding: utf-8 -*-

a, b, c = input().split(" ")
a = int(a)
b = int(b)
c = int(c)

maiorab = (a + b + abs(a - b)) / 2
maior = (maiorab + c + abs(maiorab - c)) / 2

print("%d eh o maior" %maior)

File: C/1013.c
#include <stdio.h>

int main(){
    int a, b, c, maiorAB, maior;
    
    scanf("%d", &a);
    scanf("%d", &b);
    scanf("%d", &c);
    maiorAB = (a + b + abs(a - b)) / 2;
    maior = (maiorAB + c + abs(maiorAB - c)) / 2;
    printf("%d eh o maior\n", maior);
    return 0;
}
